airflow/cli/commands/role_command.py

```diff
@@ -41,3 +39,259 @@ def roles_create(args):
     appbuilder = cached_app().appbuilder  # pylint: disable=no-member
     for role_name in args.role:
         appbuilder.sm.add_role(role_name)
+
+
+mapping = {
+    # ========================= Airflow Views
+    ("Airflow", "can_blocked"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_clear"): [("Dag", "can_read"), ("Task", "can_read"), ("TaskInstance", "can_delete")],
+    ("Airflow", "can_code"): [("Dag", "can_read"), ("DagCode", "can_read")],
+    ("Airflow", "can_dag_details"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_dag_stats"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_dagrun_clear"): [
+        ("Dag", "can_read"),
+        ("Task", "can_read"),
+        ("TaskInstance", "can_delete"),
+    ],
+    ("Airflow", "can_dagrun_failed"): [("Dag", "can_read"), ("DagRun", "can_edit")],
+    ("Airflow", "can_dagrun_success"): [("Dag", "can_read"), ("DagRun", "can_edit")],
+    ("Airflow", "can_delete"): [("Dag", "can_delete")],
+    ("Airflow", "can_duration"): [("Dag", "can_read"), ("TaskInstance", "can_read"), ("Task", "can_read")],
+    ("Airflow", "can_extra_links"): [("Dag", "can_read"), ("Task", "can_read")],
+    ("Airflow", "can_failed"): [("Dag", "can_read"), ("Task", "can_read"), ("TaskInstance", "can_edit")],
+    ("Airflow", "can_gantt"): [("Dag", "can_read"), ("Task", "can_read"), ("TaskInstance", "can_read")],
+    ("Airflow", "can_get_logs_with_metadata"): [
+        ("Dag", "can_read"),
+        ("TaskInstance", "can_read"),
+        ("Log", "can_read"),
```

Comment:
I think `Log` is a different resource -- this "get-logs" is different to the LogsModelView -- the later is "Audit Logs", this is reading the log file from task execution.